h/views/api/group_annotations.py

```diff
@@ -0,0 +1,46 @@
+from h.models import Annotation
+from h.schemas.pagination import Pagination
+from h.schemas.api.group import FilterGroupAnnotationsSchema
+from h.security import Permission
+from h.services.annotation_read import AnnotationReadService
+from h.traversal import GroupContext
+from h.views.api.config import api_config
+from h.schemas.util import validate_query_params
+
+
+@api_config(
+    versions=["v1", "v2"],
+    route_name="api.group_annotations",
+    request_method="GET",
+    link_name="group.annotations.read",
+    description="Fetch a list of all annotations of a group",
+    permission=Permission.Group.MODERATE,
```

Comment:
This endpoint is not particularly moderation oriented but at the moment is only used for that.

Scope it to the moderation permission for now.

Reply:
Does that only check if the user is a moderator "in general", or if they are a moderator in the group in question? I reckon we should check the moderator does in fact belong to the group which annotations they are trying to list.

Reply:
Yeah this checks if the current user is a moderator in the relevant course, the group on the request context, see:

https://github.com/hypothesis/h/blob/main/h/security/permission_map.py#L24